src/cache_cf.cc

```diff
@@ -968,9 +970,9 @@ configDoConfigure(void)
 #endif
         }
 #if USE_OPENSSL
-        Ssl::useSquidUntrusted(Config.ssl_client.sslContext.get());
+        Ssl::useSquidUntrusted(Config.ssl_client.sslContext->get());
```

Comment:
Technically, this code should be using defaultPeerContext.raw instead, but defaultPeerContext is initialized later. I suggest using the deprecated field here unless others insist on reordering a few lines of code to use defaultPeerContext for this useSquidUntrusted() call:

```suggestion
        Ssl::useSquidUntrusted(Config.ssl_client.sslContext_->get());
```

Reply:
Done (dc66a68).